RecoVertex/PrimaryVertexProducer_Alpaka/plugins/alpaka/PrimaryVertexProducer_Alpaka.cc

```diff
@@ -0,0 +1,166 @@
+#include "DataFormats/PortableVertex/interface/alpaka/VertexDeviceCollection.h"
+#include "DataFormats/PortableVertex/interface/VertexHostCollection.h"
+#include "DataFormats/BeamSpot/interface/BeamSpotHost.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/stream/EDProducer.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/EDPutToken.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ESGetToken.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "DataFormats/Math/interface/AlgebraicROOTObjects.h"
+
+#include "BlockAlgo.h"
+#include "ClusterizerAlgo.h"
+#include "FitterAlgo.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+  /**
+   * This class does vertexing by
+   * - consuming set of portablevertex::Track
+   * - clusterizing them into track clusters
+   * - fitting cluster properties to vertex coordinates
+   * - produces a device vertex product (portablevertex::Vertex)
+   */
+  class PrimaryVertexProducer_Alpaka : public stream::EDProducer<> {
+  public:
+    PrimaryVertexProducer_Alpaka(edm::ParameterSet const& config) {
```

Comment:
Also here the `config` needs to be passed to the base class
```suggestion
    PrimaryVertexProducer_Alpaka(edm::ParameterSet const& config): EDProducer(config) {
```